crates/samples/no_default_features/src/lib.rs

```diff
@@ -10,11 +10,26 @@
 //!
 //! This sample demonstrates it is possible to use the library with default-features = false and ensures that that scenario remains compiling as PRs go into the repository.
 //!
-use mssf_core::runtime::CodePackageActivationContext;
+
+use mssf_core::runtime::{package_change::PackageChangeEvent, CodePackageActivationContext};
 #[no_mangle]
 fn test_fn() {
-    // Make sure we link something
-    //
-    let my_ctx = CodePackageActivationContext::create();
-    my_ctx.unwrap();
+    let my_ctx = CodePackageActivationContext::create().unwrap();
+
+    // One might wish to use such a callback to e.g. trigger custom handling of configuration changes
+    // This doesn't require the config feature to be enabled
+    let _handler = my_ctx.register_config_package_change_handler( |c|
```

Comment:
U should move this to one of the example apps.

Reply:
Huh? this is one of the samples

Reply:
Oh, you mean one of the full applications.
Yeah, that could make sense.
I do want to ensure this continues to build without those features though.

Reply:
I think the logical example to add it to would be echomain

As we could demonstrate how to hook it up to trigger a Config reload

Reply:
Basically this but with our source:
https://github.com/rust-cli/config-rs/blob/main/examples/watch/main.rs

Reply:
Given I'd like to have this build here as well, can I defer that to a second PR?

Reply:
U can keep it here or add to the working app.
But note that this code here cannot run because it is not launched by SF, and we are not packaging it in an app.

Reply:
Added it to echomain as well (but haven't had a chance to test that app yet)
Right, point of this is to make sure we can package w/o tokio